packages/x/xfce4-desktop-branding/package.yml

```diff
@@ -0,0 +1,29 @@
+name       : xfce4-desktop-branding
+version    : '1.0.0'
+release    : 1
+source     :
+    - git|https://github.com/getsolus/xfce4-desktop-branding.git : dd4e61808de452a2d154580a2d38d1fec248b242
+homepage   : https://github.com/getsolus/xfce4-desktop-branding
+license    : GPL-2.0-only
+component  :
+    - desktop.xfce
+summary    :
+    - Defaults for the XFCE4 Desktop.
+description:
+    - Defaults for the XFCE4 Desktop.
+rundeps    :
+    - breeze-cursor-theme
+    - font-hack-ttf
+    - noto-sans-ttf
+    - noto-serif-ttf
+    - papirus-icon-theme
+    - qogir-gtk-theme
+    - qtstyleplugins
+    - solus-artwork
+#build      : |
+#    %make install-live
```

Comment:
This should probably be deleted if it's not used.

Reply:
It might also make sense to squash/fixup the commits with an interactive rebase, so there are only two commits in the log, where one is `xfce4-desktop-branding` and the other is `xfce4-session`?

Reply:
Yeah, probably also that.

Reply:
> This should probably be deleted if it's not used.

Good catch, I'll do that.